pkg/term/term_linux_test.go

```diff
@@ -0,0 +1,120 @@
+//+build linux
+
+package term
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
I'm more concerned that tests don't run unless you happen to provide this flag. and there's really no way to discover this flag. I really don't think this is a good solution. Skipping if `uid != 0` is the same behaviour without the need to discover a flag.

If we know a test is going to fail, why bother running it? This is what we do for all the integration tests, we skip them if the environment does not support the test. This is pretty standard for many tests suites I think.

If for some reason we really want to force these to run, unless someone sets something explicit then I think we should be using an environment variable. We can always run the tests, unless the environment variable `PKG_TERM_SKIP_ROOT_ONLY_TESTS` is set. Then we can inform users about this env var with an error  message when `uid != 0`.